.flake8

```diff
@@ -0,0 +1,9 @@
+[flake8]
```

Comment:
Shouldn't this file be removed? So flake8 will only be configured using pyproject.toml

Reply:
Minor comments